gnosis/safe/api/transaction_service_api.py

```diff
@@ -275,3 +275,74 @@ def post_transaction(self, safe_tx: SafeTx) -> bool:
         if not response.ok:
             raise SafeAPIException(f"Error posting transaction: {response.content}")
         return True
+
+    def post_message(
+        self,
+        safe_address: ChecksumAddress,
+        message: Union[str, Dict],
+        signature: bytes,
+        safe_app_id: Optional[int] = 0,
+    ) -> bool:
+        """
+        Create safe message on transaction service for provided Safe address
+
+        :param safe_address:
+        :param message:
```

Comment:
Explain that if message is a `str` it will be encoded using EIP191, and if it's a dictionary it will be encoded using EIP712